src/components/no-results-message/noResultsMessage.css

```
@@ -0,0 +1,33 @@
+@import '@folio/stripes-components/lib/variables';
+
+.noResultsMessage {
+  padding: 15px;
+  color: var(--color-text-p2);
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

`top`, `bottom`, `left`, `right`, properties are used for positioning, not for size of element. In my opinion is better to use `height` and `width` for it

```suggestion
  top: 0;
  width: 100%;
  height: 100%;
```

---

@Vladyslav-Velytskyi, considering all this stuff with dynamic list height calculations + absolute positioning, I'm not sure if that will work, but we can try.